Add issue template for PEtab specification change requests (#455)
PEtab-dev/PEtab: .github/ISSUE_TEMPLATE/petab-specification-changes.md

---
name: PEtab specification changes
about: Suggest a change to the PEtab specification (beyond fixes / clarifications)
title: ''
labels: file format
assignees: ''

---

**Which problem would you like to address? Please describe.**

> A clear and concise description of which use case you want to address and if applicable, why the current specifications, don't fulfill your requirements.

Reviewer: ```suggestion
A clear and concise description of which use case you want to address and if applicable, why the current specifications do not fulfill your requirements.
```

> **Describe the solution you'd like**

Reviewer: ```suggestion
**Describe the solution you would like**
```

> A clear and concise description of what you want to happen. Please describe any additional fields / files you would want to add, including allowed inputs and implications.

Reviewer: ```suggestion
A clear and concise description of the changes you want to propose. Please describe any additional fields / files you would want to add, including allowed inputs and implications.
```

> **Describe alternatives you've considered**

Reviewer: ```suggestion
**Describe alternatives you have considered**
```

> A clear and concise description of any alternative solutions or features you've considered.

Reviewer: ```suggestion
A clear and concise description of any alternative solutions or features you have considered.
```
